clarity/src/vm/database/clarity_db.rs

```diff
@@ -187,6 +192,14 @@ impl HeadersDB for &dyn HeadersDB {
 }
 
 impl BurnStateDB for &dyn BurnStateDB {
```

Comment:
This entire `impl` block is unnecessary. If you delete it, the build will fail, but just remove some of the extra `&`s to fix it.

Same for `impl HeadersBD` above

Reply:
Good call. Removed in 9e270b7934eaa35522e88349e9d7608e39346fd7.